test/aave-v3/TestSupplyVault.t.sol

```diff
@@ -169,7 +169,7 @@ contract TestSupplyVault is TestSetupVaults {
         uint256 balanceBefore = vaultSupplier1.balanceOf(rewardToken);
 
         (address[] memory rewardTokens, uint256[] memory claimedAmounts) = daiSupplyVault
-            .claimRewards(address(vaultSupplier1));
```

Comment:
hu weird no?

Reply:
Yeah, I'm not sure what to make of this nor how to fix it since everything autoformats.

Reply:
I've the same in local but it was different in the past, right?

Reply:
I don't think it was different, it's the same format as on morpho-v1?

Reply:
Ah yes you're right. But then I don't understand why it was not like that before...